### PR TITLE
testing PR

### PR DESCRIPTION
**Pull request by pattyr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -217,4 +217,5 @@ Happy Hacking!
 
 ## Copyright & License
 * Copyright (c) 2017 Juan Manuel Torres, Katherine Marino, Daniel Lee
-* MIT License
\ No newline at end of file
+* MIT License
+˚
```
Comment (onema) [inline README.md]: Can you please remove this!

Comment (pattyr) [reply README.md]: ok

File: README.md [hunk 2]
```diff
@@ -217,4 +217,4 @@ Happy Hacking!
 
 ## Copyright & License
 * Copyright (c) 2017 Juan Manuel Torres, Katherine Marino, Daniel Lee
-* MIT License
\ No newline at end of file
+* MIT License
```
Comment (onema) [inline README.md]: Now that I think about it, you need add the degree

Comment (pattyr) [reply README.md]: make up your mind!

File: README.md (hunk 1 above)
Comment (onema) [reply README.md]: Sigh... this still doesn't look good. Please remove it!

**Pull request merged**